<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>832</width>
        <height>978</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="baseStationsTab">
          <attribute name="icon">
           <iconset resource="../res/res.qrc">
            <normaloff>:/icons/ui/station.png</normaloff>:/icons/ui/station.png</iconset>
          </attribute>
          <attribute name="title">
           <string>Base Stations</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="BaseStationsWidget" name="baseStationsWidget" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="usersTab">
          <attribute name="icon">
           <iconset resource="../res/res.qrc">
            <normaloff>:/icons/ui/users.png</normaloff>:/icons/ui/users.png</iconset>
          </attribute>
          <attribute name="title">
           <string>Users</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="UsersWidget" name="usersWidget" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="reportsTab">
          <attribute name="icon">
           <iconset resource="../res/res.qrc">
            <normaloff>:/icons/ui/report.png</normaloff>:/icons/ui/report.png</iconset>
          </attribute>
          <attribute name="title">
           <string>Reports</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="ReportsWidget" name="reportsWidget" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="emailTab">
          <attribute name="icon">
           <iconset resource="../res/res.qrc">
            <normaloff>:/icons/ui/email.png</normaloff>:/icons/ui/email.png</iconset>
          </attribute>
          <attribute name="title">
           <string>Email</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="verticalSpacing">
               <number>6</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>SMTP server address</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Username</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" colspan="2">
               <widget class="QSpinBox" name="emailPort">
                <property name="maximum">
                 <number>65535</number>
                </property>
                <property name="value">
                 <number>465</number>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Password</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>From field</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Recipients</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>SMTP Port</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QComboBox" name="emailConnection">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <item>
                 <property name="text">
                  <string>SSL</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>TLS</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Standard (Unencrypted)</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="1" colspan="3">
               <widget class="QLineEdit" name="emailHost">
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>smtp.gmail.com</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1" colspan="3">
               <widget class="QLineEdit" name="emailUsername">
                <property name="placeholderText">
                 <string>you@company.com</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1" colspan="3">
               <widget class="QLineEdit" name="emailPassword">
                <property name="echoMode">
                 <enum>QLineEdit::PasswordEchoOnEdit</enum>
                </property>
               </widget>
              </item>
              <item row="4" column="1" colspan="3">
               <widget class="QLineEdit" name="emailFrom">
                <property name="text">
                 <string>Sense Monitoring System</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1" colspan="3">
               <widget class="QListWidget" name="emailRecipientList"/>
              </item>
              <item row="6" column="1" colspan="3">
               <widget class="QWidget" name="widget_8" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="emailAddRecipient">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="../res/res.qrc">
                     <normaloff>:/icons/ui/add.png</normaloff>:/icons/ui/add.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>24</width>
                     <height>24</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="emailRemoveRecipient">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="../res/res.qrc">
                     <normaloff>:/icons/ui/error.png</normaloff>:/icons/ui/error.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>24</width>
                     <height>24</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>516</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="emailTest">
                <property name="text">
                 <string>Test</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="emailApply">
                <property name="text">
                 <string>Apply</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="emailReset">
                <property name="text">
                 <string>Reset</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="ftpTab">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <attribute name="icon">
           <iconset resource="../res/res.qrc">
            <normaloff>:/icons/ui/ftp.png</normaloff>:/icons/ui/ftp.png</iconset>
          </attribute>
          <attribute name="title">
           <string>FTP</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QGridLayout" name="gridLayout">
              <property name="verticalSpacing">
               <number>6</number>
              </property>
              <item row="0" column="1" colspan="2">
               <widget class="QLineEdit" name="ftpHost">
                <property name="text">
                 <string>ftp://server.com</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Folder</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="ftpFolder"/>
              </item>
              <item row="5" column="0">
               <widget class="QCheckBox" name="ftpUploadBackups">
                <property name="text">
                 <string>Upload Backups Every</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Username</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Server address</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Password</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1" colspan="2">
               <widget class="QLineEdit" name="ftpUsername"/>
              </item>
              <item row="3" column="1" colspan="2">
               <widget class="QLineEdit" name="ftpPassword">
                <property name="echoMode">
                 <enum>QLineEdit::PasswordEchoOnEdit</enum>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QPushButton" name="ftpBrowse">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../res/res.qrc">
                  <normaloff>:/icons/ui/ftp.png</normaloff>:/icons/ui/ftp.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="5" column="1" colspan="2">
               <widget class="QSpinBox" name="ftpBackupPeriod">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="suffix">
                 <string> day(s)</string>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>99999</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Server port</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" colspan="2">
               <widget class="QSpinBox" name="ftpPort">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>65535</number>
                </property>
                <property name="value">
                 <number>21</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="ftpTest">
                <property name="text">
                 <string>Test</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="ftpApply">
                <property name="text">
                 <string>Apply</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="ftpReset">
                <property name="text">
                 <string>Reset</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="sensorsTab">
          <attribute name="icon">
           <iconset resource="../res/res.qrc">
            <normaloff>:/icons/ui/sensor.png</normaloff>:/icons/ui/sensor.png</iconset>
          </attribute>
          <attribute name="title">
           <string>Sensors</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QWidget" name="widget_6" native="true">
             <layout class="QFormLayout" name="formLayout_5">
              <property name="verticalSpacing">
               <number>6</number>
              </property>
              <item row="4" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Actual Comms Period</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QDoubleSpinBox" name="sensorsComputedCommsPeriod">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="suffix">
                 <string> minutes</string>
                </property>
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="minimum">
                 <double>1.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="5" column="0" colspan="2">
               <widget class="QCheckBox" name="sensorsSleeping">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Put Sensors To Sleep</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0" colspan="2">
               <widget class="QLabel" name="label_16">
                <property name="font">
                 <font>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>To put them to sleep, check the box and press the sensor button briefly</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0" colspan="2">
               <widget class="QLabel" name="label_17">
                <property name="font">
                 <font>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>To wake them up, uncheck the box and press the sensor button briefly</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Measurement Period</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="sensorsMeasurementPeriod">
                <property name="suffix">
                 <string> minutes</string>
                </property>
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="minimum">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>5.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Comms Period</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="sensorsCommsPeriod">
                <property name="suffix">
                 <string> minutes</string>
                </property>
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="minimum">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>15.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QWidget" name="widget_7" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="sensorsApply">
                <property name="text">
                 <string>Apply</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="sensorsReset">
                <property name="text">
                 <string>Reset</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseStationsWidget</class>
   <extends>QWidget</extends>
   <header>BaseStationsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReportsWidget</class>
   <extends>QWidget</extends>
   <header>ReportsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UsersWidget</class>
   <extends>QWidget</extends>
   <header>UsersWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>emailAddRecipient</tabstop>
  <tabstop>emailRemoveRecipient</tabstop>
  <tabstop>emailTest</tabstop>
  <tabstop>emailApply</tabstop>
  <tabstop>emailReset</tabstop>
  <tabstop>ftpHost</tabstop>
  <tabstop>ftpUsername</tabstop>
  <tabstop>ftpPassword</tabstop>
  <tabstop>ftpFolder</tabstop>
  <tabstop>ftpUploadBackups</tabstop>
  <tabstop>ftpBackupPeriod</tabstop>
  <tabstop>ftpApply</tabstop>
  <tabstop>ftpReset</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>sensorsComputedCommsPeriod</tabstop>
  <tabstop>sensorsSleeping</tabstop>
  <tabstop>sensorsApply</tabstop>
  <tabstop>sensorsReset</tabstop>
 </tabstops>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ftpUploadBackups</sender>
   <signal>toggled(bool)</signal>
   <receiver>ftpBackupPeriod</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>659</x>
     <y>513</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
